benefits/core/migrations/0002_sample_data.py

```diff
@@ -146,9 +146,11 @@ def load_sample_data(app, *args, **kwargs):
         form_content_title=_("eligibility.pages.confirm.mst.content_title"),
         form_blurb=_("eligibility.pages.confirm.mst.p[0]"),
         form_sub_label=_("eligibility.forms.confirm.mst.fields.sub"),
-        form_sub_placeholder="B1234567",
+        form_sub_help_text=_("eligibility.forms.confirm.mst.fields.sub.help_text"),
+        form_sub_placeholder="TODO: B1234567",
```

Comment:
I think we can make this `#12345` based on design discussions

Reply:
But this raises an interesting question - we don't have the `#` symbol in the data we'll get from Velocity. So we may need to strip it out here before sending to the Server?